### ISSUE TITLE
rgba()/hsl()/hsla() on CSSLint v0.8.1

### ISSUE THREAD

**Issue by hail2u** [CSSLint/csslint#206]
Using rgba()/hsl()/hsla() in CSS code, CSSLint v0.8.1 warns like this:

```
Expected color or one of (inherit) but found 'rgba(0 , 0 , 0 , 0.5)'.
```

Tested on csslint.net with following code:

``` css
.rgb {
  color: rgb(0, 0, 0);
}

.rgba {
  color: rgba(0, 0, 0, 0.5);
}

.hsl {
  color: hsl(0, 0%, 0%);
}

.hsla {
  color: hsla(0, 0%, 0%, 0.5);
}
```
